python/lsst/ts/MTMount/mtmount_csc.py

```diff
@@ -645,6 +620,12 @@ async def send_commands(self, *commands, do_lock=True):
         except ConnectionResetError:
             if future is not None:
                 future.setnoack("Connection lost")
+        except salobj.ExpectedError as e:
```

Comment:
Again, seems like an awful lot of work just to change the log message from `exception` to `error`. I think I know why you are doing it so ok.

Reply:
I reviewed all instances and changed a few where I didn't think the exception was interesting. I agree it makes for more duplication than I like. A command should normally fail with ExceptedError (e.g. if it is rejected or times out), so I think a traceback may prove useful if that's not the error raised.